### DISCUSSION TITLE
Menu editor Item Visibility

### DISCUSSION THREAD

**Discussion by Gpapidas** [avo-hq/avo#1239] [Help] [answered]
Hello everyone
I have a question regarding Item visibility in menu editor ( https://docs.avohq.io/2.0/menu-editor.html?fbclid=IwAR1wjehKjW8kQyaK1RS_ImINX1GGYUczZ2rfAqYp-breMMh7A5FjtSzRKzg#item-visibility )
Is this a pro only feature?
I have implemented this setting and while it works on localhost, it does not on production.
I am using a community license, is this the issue or should I be looking for something else happening there?

**Comment by adrianthedev** [comment, marked as answer]
Hey @Gpapidas. Please have a look at the header of the page. On the new docs, we added the license requirements on top of the page so you can see pretty quick if they are a part of the Community or Pro license. Some features have most of the options under Community and some options under Pro, but they are signaled properly.

![CleanShot 2022-09-17 at 14 17 51@2x](https://user-images.githubusercontent.com/1334409/190856330-9b95a901-312f-41cb-b2ef-70d71ec7864d.jpg)

**Comment by Gpapidas** [comment]
Hey, thank you for the answer, I had not noticed the label :)
